hackathon_site/registration/models.py

```diff
@@ -25,23 +28,94 @@ def __str__(self):
 
 
 class Application(models.Model):
-    user = models.ForeignKey(User, on_delete=models.CASCADE, null=False)
-    team = models.ForeignKey(Team, on_delete=models.CASCADE, null=False)
+    GENDER_CHOICES = [
+        (None, ""),
+        ("male", "Male"),
+        ("female", "Female"),
+        ("non-binary", "Non-binary"),
+        ("other", "Other"),
+        ("no-answer", "Prefer not to answer"),
+    ]
+
+    ETHNICITY_CHOICES = [
+        (None, ""),
+        ("american-native", "American Indian or Alaskan Native"),
+        ("asian-pacific-islander", "Asian / Pacific Islander"),
+        ("black-african-american", "Black or African American"),
+        ("hispanic", "Hispanic"),
+        ("caucasian", "White / Caucasian"),
+        ("other", "Multiple ethnicity / Other"),
+        ("no-answer", "Prefer not to answer"),
+    ]
+
+    STUDY_LEVEL_CHOICES = [
+        (None, ""),
+        ("highschool", "High School"),
+        ("undergraduate", "Undergraduate"),
+        ("gradschool", "Graduate School"),
+        ("other", "Other"),
+    ]
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE, null=False)
+    team = models.ForeignKey(
+        Team, related_name="applications", on_delete=models.CASCADE, null=False
+    )
 
     # User Submitted Fields
-    preferred_name = models.CharField(max_length=255, null=False)
-    birthday = models.DateField(auto_now=False, auto_now_add=False)
-    gender = models.CharField(max_length=50, null=False)
-    phone_number = models.CharField(max_length=20, null=False)
-    resume = models.FileField(upload_to="applications/resumes/", null=False)
-    q1 = models.TextField(null=False)
-    q2 = models.TextField(null=False)
-    q3 = models.TextField(null=False)
-    mlh_conduct_agree = models.BooleanField(default=False, null=False)
-    mlh_data_agree = models.BooleanField(default=False, null=False)
+    birthday = models.DateField(null=False)
+    gender = models.CharField(max_length=50, choices=GENDER_CHOICES, null=False)
+    ethnicity = models.CharField(max_length=50, choices=ETHNICITY_CHOICES, null=False)
+    phone_number = models.CharField(
+        max_length=20,
+        null=False,
+        validators=[
+            validators.RegexValidator(
+                r"^(?:\+\d{1,2})?\s?\(?\d{3}\)?[\s-]?\d{3}[\s-]?\d{4}$",
+                message="Enter a valid phone number.",
```

Comment:
We should display on the form an example of a valid phone number so they know how we expect it